### ISSUE TITLE
Regression tests need updating for 2.0.1

### ISSUE THREAD

**Issue by mbanck** [2ndQuadrant/pglogical#123]
Several tests in `sql/` have `2.0.0` as version hardcoded in, which no longer works since `2.0.1` is out:

```
pglogical-2.0.1$ grep VERSION..2.0.0 sql/*
sql/init_fail.sql:CREATE EXTENSION IF NOT EXISTS pglogical VERSION '2.0.0';
sql/multiple_upstreams.sql:CREATE EXTENSION IF NOT EXISTS pglogical VERSION '2.0.0';
sql/node_origin_cascade.sql:CREATE EXTENSION IF NOT EXISTS pglogical VERSION '2.0.0';
```

**Comment by mbanck** [comment]
Ok, so this got fixed in 30f7454, but no release has been rolled since then. Also, the `REL_2_0_STABLE` branch seems to be lacking the v10 compatibilty from master, is this going to be cherry-picked and release soon as well?

**Comment by mbanck** [comment]
Well, closing this as it's fixed on the `REL2_0_STABLE` branch
